index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="mystyle.css">
+  </head>
+  <body>
+
+    <div id="banner">
+      <ul id="banban">
+        <li id="user-name"></li>
+        <li><a id="done" href = "logout.php">logout</a></li>
+      </ul>
+    </div>
+    <div class="header">
+      <h1>3/4D's homepage</h1>
+      <p>Choose your game below:</p>
+      <form method="link" action="ttupdate.php">
+        <input type = "submit" value = "magic timestable button">
+      </form>
+      <form method="link" action="divupdate.php">
+        <input type = "submit" value = "magic division button">
+      </form>
+      <form method="link" action="doubles.php">
+        <input type = "submit" value = "magic doubles button">
+      </form>
+      <form method="link" action="targettraining.php">
+        <input type = "submit" value = "Target Practice">
+      </form>
+      <button id="target">Set Target</button>
+      <div class="input-group">
+        <input type="number" id="input" size = 4 autofocus autocomplete="off" class="form-control"/>
+        <input type="button" id="fireButton" value = "submit"/>
+      </div>
+    </div>
+    <div></div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="jquery-ui.js"></script>
+    <script>
+
+    // check user session
```

Comment:
Check user session.
Every pages need to do it.
TODO : To provide offline modes, we need to save session data from server in device local store.